include/openssl/asn1.h

```diff
@@ -141,6 +141,8 @@ DEFINE_STACK_OF(X509_ALGOR)
 # define ASN1_STRING_FLAG_MSTRING 0x040
 /* String is embedded and only content should be freed */
 # define ASN1_STRING_FLAG_EMBED 0x080
+/* String should be parsed in RFC 5280's time format */
+# define ASN1_STRING_FLAG_X509_TIME 0x100
```

Comment:
I've been wondering if we shouldn't just set this flag somehow instead of creating a new API just to set this flag. I don't actually know enough about the whole API to know how to extend it best, so I'm open for other ideas.